serving/docker/partition/partition.py

```diff
@@ -202,7 +208,9 @@ def run_partition(self) -> str:
             self.cleanup()
             return partition_stdout
         else:
-            raise Exception("Partitioning was not successful.")
+            logging.error(
+                f"Partitioning was not successful: {partition_stderr}")
+            raise Exception(partition_stderr)
```

Comment:
Is this the final customer facing Exception name ?

Reply:
It looks like this: 
```
CompilationFatalError('ValueError: Model partitioning does not support split model artifacts. Use normal model artifacts and rerun.\\n')
```

(Will push commit replacing CompilationFatalError with OptimizationFatalError as discussed)